packages/mermaid/src/docs/.vitepress/scripts/fetch-avatars.ts

```diff
@@ -18,18 +18,21 @@ async function download(url: string, fileName: URL) {
     const image = await fetch(url);
     await writeFile(fileName, Buffer.from(await image.arrayBuffer()));
   } catch (error) {
-    console.error(error);
+    console.error('failed to load', url, error);
   }
 }
 
 async function fetchAvatars() {
   await mkdir(fileURLToPath(new URL(getAvatarPath('none'))).replace('none.png', ''), {
     recursive: true,
   });
+
   contributors = JSON.parse(await readFile(pathContributors, { encoding: 'utf-8' }));
-  for (const name of contributors) {
-    await download(`https://github.com/${name}.png?size=100`, getAvatarPath(name));
-  }
+  let avatars = contributors.map((name) => {
+    download(`https://github.com/${name}.png?size=100`, getAvatarPath(name));
+  });
+
+  await Promise.all(avatars);
```

Comment:
I switched to sequential download because of GitHub's API rate limits. 

Let's see if this will cause issues again. 🤞

Reply:
What is their API rate? We can cut it down to a minimum which is allowed, so to say, split avatars in groups of 10

Reply:
Let's do that once it breaks.